scipy/stats/_continuous_distns.py

```diff
@@ -905,9 +905,10 @@ class chi_gen(rv_continuous):
 
     .. math::
 
-        f(x, df) = \frac{x^{df-1} \exp(-x^2/2)}{2^{df/2-1} \gamma(df/2)}
+        f(x; df) = \frac{x^{df-1} \exp(-x^2/2)}{2^{df/2-1} \Gamma(df/2)}
```

Comment:
Why the change `f(x, df)` -> `f(x; df)`? I.e. comma -> semi-colon?

Reply:
I think it ist better to seperate the shape parameters with a semicolon (Most common notation is probably to use subscripts like `f_\beta\) , see e.g. the incomplete beta function here https://en.m.wikipedia.org/wiki/Beta_function

However, one should probably agree on a consistent approach for all distributions here...

Reply:
It wouldn't be an unreasonable distinction. However only two of the existing PDF functions (`norminvgauss` and `t`) use semi-colon, and the other 63 use comma. So probably better to keep this one as semi-colon.
[Even the referenced incomplete beta is inconsistent here.  The regularized incomplete beta function is written`$I_x(a, b)$` with the shape parameters inside the brackets, but the `x` is now a subscript!]

Reply:
agree, i changed the semicolon to comma again.